python/tk_hiero_export/collating_exporter_ui.py

```diff
@@ -39,6 +39,18 @@ def populateUI(self, widget, properties=None):
         self._collateNameProperty = UIPropertyFactory.create(type(value), key=key, value=value, dictionary=properties, label=label, tooltip=collateShotNameToolTip)
         layout.addRow(label, self._collateNameProperty)
 
+        if cut_support:
+            cut_lbl = PySide.QtGui.QLabel(
+                "NOTE: Turning on collating will disable Cut/CutItem creation "
+                "in Shotgun."
+            )
```

Comment:
I think this comes off as a little technical and we should try to avoid the passive voice.
How about ```(NOTE: Cuts in Shotgun are only created when collate is off.)``` or something like that?